spec/features/events/new_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+module Features
+  describe 'fulfil order' do
+    let(:new_event_page) do
+      NewEventPage.new(
+        location: location,
+        name: name,
+        takes_place_on: takes_place_on,
+      )
+    end
+
+    let(:email) { user.email }
+    let(:name) { "Maker's Market" }
+    let(:location) { "Sandbach" }
+    let(:password) { user.password }
+    let(:signin_page) { SigninPage.new(email, password) }
+    let(:takes_place_on) { "12 October 2015" }
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'sends an email to the customer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.